src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -19,13 +21,24 @@
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 import java.io.IOException;
 
+/**
+ * This class calls endpoint to extract metadata.
+ */
 @RestController
 @RequestMapping("/api")
 public class ExtractMetadataController {
     private static final Logger LOGGER = LogManager.getLogger();
 
     /**
-     * Endpoint for extract metadata
+     * Creates endpoint to extract metadata.
+     *
+     * @param fileToExtract is a file to extract.
+     * @param isMoreInfo is more information about the file.
+     * @param nameExport is a name of file to export.
+     * @param format is the format to file.
+     * @return path to download files.
+     * @throws IllegalStateException is a exception if process is Illegal.
+     * @throws IOException is a exception when invalid input is provided.

```

Comment:
Change name of method to metadataController or something like that

Reply:
or extractMetadataController

Reply:
mmm ok

Reply:
but This PR is oriented to correct code convetions the refactor ill be made in the next PRs